### DISCUSSION TITLE
Does supabase auth allow multiple login from the same account?

### DISCUSSION THREAD

**Discussion by tamlahtoo** [supabase/supabase#2194] [General]
Can the same user login on multiple different device, say 100 devices at the same time? (if can, what is the number?)
Also Firebase offer the technology that the user doesn't need to sign in (by storing user_info on with Firebase SDK on device).
Does supabase SDK (especially Flutter SDK) does it itself or do I need to store authentication token and check the logn in status myself?

**Comment by ashleyconnor** [comment]
Concurrent sessions works. I don't think there's an upper limit. I don't know about the Flutter SDK so I'll let someone else answer that question.
